packages/nx/src/daemon/server/server.ts

```diff
@@ -352,9 +361,12 @@ export async function handleResult(
 }
 
 function handleInactivityTimeout() {
-  if (hasRegisteredFileWatcherSockets()) {
+  if (
+    hasRegisteredFileWatcherSockets() ||
+    hasRegisteredProjectGraphListenerSockets()
```

Comment:
This will keep the daemon alive, presumably as long as Nx console is alive - is that what we want? If so, that's fine.